### ISSUE TITLE
API error: 500 

### ISSUE THREAD

**Issue by emddummi** [vas3k/home-assistant-berlin-transport#39]
Hi,

i cant get it working, is something wrong, where i can find information for you?

`API error: 500 Server Error: Internal Server Error for url: https://v6.vbb.transport.rest/stops/900160016/departures?when=2024-11-15T13%3A30%3A04.031331&direction=S+Storkower+Str.+%28Berlin%29&duration=30&results=15&suburban=False&subway=False&tram=False&bus=True&ferry=False&express=False&regional=False`

**Comment by vas3k** [comment]
Well, `500 Server Error` means that BVG server is temporary down. There's not much we can do here to help them :)

For me it's up again now

**Comment by emddummi** [comment]
Erstmals aufgetreten: 14. November 2024 um 13:54:31 (1043 Vorkommnisse)

Since my first Installation...

**Comment by emddummi** [comment]
I paste my link in Browser and vbb respond with:

direction must be an IBNR

**Comment by ChristianFeldmann** [comment]
If anyone stumples on this again: The direction must also be a station ID and not a raw name. You can directly use your browser to query the API:

> 'https://v6.vbb.transport.rest/locations?query=Storkower&results=1'

Then use this ID (in this case 900110012) as the 'Direction'.

**Comment by Cornelicorn** [comment]
Yes, the direction has to be the stop id for now, I opened #77 as an enhancement issue to add the search for directions directly into the configuration flow.
